### PR TITLE
Added favicon

### LINKED ISSUE #228: 🧑‍💻: [Enhancement] : Adding favicon in website
:red_circle: **Title** : Adding favicon
:red_circle: **Enhancement Aim** : Making it look good to user
:red_circle: **Changes** : I want to add a favicon to the website

### Screenshots 📷
image

image

We can add a chanakya icon as a favicon which will give a good look to our website , I think users will like this feature

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : Maana Ajmera
- What is your participant role? GSSOC
***********************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

### PR DESCRIPTION
**Pull request by Maana-Ajmera**
Closes #228 
- **Title** : Favicon
- **Name:** : Maana Ajmera
- **Idenitfy yourself:** GSSOC

### Describe the add-ons or changes you've made 📃
I have added the favicon to the website which adds a good look to it

### Type of change ☑️

What sort of change have you made:

- [X] New feature (non-breaking change which adds functionality)
- [X] Code style update (formatting, local variables)

### Checklist: ☑️

- [X] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/Chanakya-Niti/blob/main/README.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/Chanakya-Niti/blob/main/CODE_OF_CONDUCT.md) of this project.
- [X] This PR does not contain plagiarized content.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly wherever it was hard to understand.
- [X] My changes generate no new warnings.

### Screenshots 📷

image

### CODE REVIEW

File: favicon.png
Comment (Avdhesh-Varshney) [inline favicon.png:1]: convert it into ico format

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

File: favicon.png
Comment (Avdhesh-Varshney) [reply favicon.png:1]: And keep this image in circular cropping

**Comment by Maana-Ajmera** [comment]
image

@Avdhesh-Varshney done

**Comment by Maana-Ajmera** [comment]
@Avdhesh-Varshney I have resolved conflicts and done the changes , please review my PR and if there is anything else that needs to be changes , please let me know

**Comment by Avdhesh-Varshney** [comment]
@Maana-Ajmera it is resolved.

**Pull request closed without merging**